pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/rule/AbstractApexRule.java

```diff
@@ -101,13 +101,16 @@
 
 public abstract class AbstractApexRule extends AbstractRule
         implements ApexParserVisitor, ImmutableLanguage, CodeClimateRule {
+	
+	private static final IntegerProperty CODACY_TIME_TO_FIX = new IntegerProperty("codacy_minutes_to_fix", 
+			"Codacy.com Time to fix in minutes", Integer.MIN_VALUE, Integer.MAX_VALUE, 5, 1.0f);
```

Comment:
5 min is the current default used by Codacy. I think its fair to use it. I changed the other things already.